### DISCUSSION TITLE
Github pages website not getting updated

### DISCUSSION THREAD

**Discussion by ghost** [community/community#39008] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello Experts,

Its almost 2 hours since I pushed my files in my repo but my website is not getting updated.
My website URL is https://sndn.github.io/

Please help.
TIA.

**Comment by airtower-luna** [comment]
The Jekyll build failed, a theme seems to be missing: https://github.com/sndn/sndn.github.io/actions/runs/3456090439/jobs/5768581165#step:4:11

You can find logs via the Actions tab on the repository page.

**Comment by subidit** [comment]
it says `theme not found`.
try uploading the whole thing again.
